extensions/components/com_redcore/admin/views/webservice/tmpl/default.php

```diff
@@ -14,150 +14,12 @@
 JHtml::_('behavior.keepalive');
 JHtml::_('rbootstrap.tooltip');
 JHtml::_('rjquery.chosen', 'select');
+JHtml::script(JUri::root() . '/media/redcore/js/redcore.js');
```

Comment:
The file is already in that folder. So I guess maybe I need to use redcore.mini.js?